101-app-configuration-store-kv/azuredeploy.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "configStoreName": {
+            "type": "string",
+            "metadata": {
+                "description": "Specifies the name of the App Configuration store."
+            }
+        },
+        "keyValueName": {
+            "type": "string",
+            "defaultValue": "myKey$myLabel",
+            "metadata": {
+                "description": "Specifies the name of the key-value resource. The name is a combination of key and label with $ as delimiter."
```

Comment:
is this syntax required (I assume deployment fails if it is and not provided?)

Reply:
It's optional. Below is the full explanation from the doc to be published. 

```
The `keyValues` resource's name is a combination of key and label. The key and label are joined by the `$` delimiter. The label is optional. In the above example, the `keyValues` resource name is `myKey$myLabel`. To create the same key-value without a label, use a key-value resource name of `myKey`.
```

Reply:
Let me know if `The label is optional.`  should be added to the description as well.

Reply:
Yes, please... assume that when the label is not provided that the $ is also not provided?

Reply:
`myKey$` has same result as `myKey`. Both mean no label.